packages/gui/src/components/plotNFT/select/PlotNFTSelectPool.tsx

```diff
@@ -107,6 +111,19 @@ const PlotNFTSelectPool = forwardRef((props: Props, ref) => {
   }));
 
   async function handleSubmit(data: FormData) {
+    if (nfts?.length > 10) {
+       createNFT = await openDialog(
+        <ConfirmDialog
+          title="Too Many NFT's"
+          confirmColor="danger"
+        >
+          <Trans>
+            You already have more then 10 PlotNFT's, you really shouldn't make more.
```

Comment:
Not thrilled with the wording, but I don't have a great alternative at the moment.

```suggestion
            You already have more than 10 Plot NFTs. Click OK if you're sure you want to create a new one.
```